daemon/seccomp_disabled.go

```diff
@@ -3,10 +3,15 @@
 package daemon
 
 import (
+	"fmt"
+
 	"github.com/docker/docker/container"
 	"github.com/opencontainers/specs/specs-go"
 )
 
 func setSeccomp(daemon *Daemon, rs *specs.Spec, c *container.Container) error {
+	if c.SeccompProfile != "" && c.SeccompProfile != "unconfined" {
+		return fmt.Errorf("Seccomp is not enabled in your Docker daemon, cannot run a custom seccomp profile.")
```

Comment:
Should we make this "available" in stead of "enabled", otherwise people may start searching for a way to *enable*? Thinking of some variations

```
this daemon does not support seccomp profiles

seccomp profiles are not supported on this daemon, you cannot specify a custom seccomp profile
```

(note: I think the convention for errors in Go is to start with a lowercase, and no period at the end)